src/app/shared/utils/utils.ts

```diff
@@ -449,8 +457,9 @@ function parseAnswerListItem(item: any) {
   if (typeof item === "string") {
     const stringProperties = item.split("|");
     stringProperties.forEach((s) => {
-      const [field, value] = s.split(":").map((v) => v.trim());
+      let [field, value] = s.split(":").map((v) => v.trim());
       if (field && value) {
+        if (["undefined", "NaN", "null", '""'].includes(value)) value = undefined;
```

Comment:
Yeh I don't think this was sensible, I've updated to only convert the string `"undefined"` to `undefined`. It is supposed to catch precisely the case where the value is `undefined` because the answer list is being generated from a data list which may not have a value in the relevant column